core/ecschema-metadata/src/SchemaFormatsProvider.ts

```diff
@@ -110,7 +117,13 @@ export class SchemaFormatsProvider implements FormatsProvider {
   public async getFormat(name: string): Promise<SchemaItemFormatProps | undefined> {
     const [schemaName, schemaItemName] = SchemaItem.parseFullName(name);
     const schemaKey = new SchemaKey(schemaName);
-    const schema = await this._context.getSchema(schemaKey);
+    let schema: Schema | undefined;
+    try {
+      schema = await this._context.getSchema(schemaKey);
```

Comment:
Likewise, getSchema will return undefined. What errors are we catching here?

Reply:
same explanation as in the reply above